.github/workflows/release-dev.yml

```
@@ -23,3 +23,25 @@ jobs:
           echo "::set-output name=json::$content"
       - run: npm version --no-git-tag-version ${{ fromJson(steps.pkg.outputs.json).version }}-$(git rev-parse --short HEAD)+${{ github.run_id }}.${{ github.run_attempt }}
       - run: npm publish --tag dev
+      - run: |
+          git tag --force dev
+          git push --force origin dev
```

I would prefer we not force push. This might be safer:

```suggestion
          git tag --force dev
          git push origin :refs/tags/dev
          git push origin --tags
```

---

Oh, I see, you delete the remote tag and then push the tag.
Nice !